spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/README.adoc

```diff
@@ -8,25 +8,25 @@ link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-data-spanner[Sprin
 
 1. Make sure that you have the Cloud SDK configured by following https://cloud.google.com/sdk/docs/[these instructions].
```

Comment:
Cloud SDK isn't mandatory to use Spring Data Spanner. I'd remove this bullet completely, then later on, for the credentials bit, mention that a user can use Cloud SDK to authenticate with application default credentials.